ui/package.json

```diff
@@ -59,25 +59,25 @@
     "react-router": "^6.2.1",
     "react-router-bootstrap": "^0.26.1",
     "react-router-dom": "^6.2.2",
-    "react-scripts": "5.0.0",
     "react-slick": "^0.29.0",
     "react-sticky": "^6.0.3",
     "redux": "4.1.2",
     "redux-form": "^8.3.8",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.4.1",
-    "sass": "^1.51.0",
     "shortid": "^2.2.6",
     "slick-carousel": "^1.8.1",
     "socket.io-client": "^4.6.1",
     "underscore": "^1.8.3",
-    "webpack": "^5.69.1",
-    "webpack-dev-server": "^4.8.0",
     "ws": "^8.5.0"
   },
   "devDependencies": {
     "@fortawesome/fontawesome-free": "^5.15.4",
-    "@testing-library/cypress": "9.0.0",
+    "@testing-library/cypress": "10.0.1",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "babel-preset-react-app": "10.0.1",
```

Comment:
Missing dep required for ESLint (and our linting config) to work properly when installed with pnpm